project/Dependencies.scala

```diff
@@ -11,8 +11,8 @@ import scala.language.implicitConversions
 object Dependencies {
   import DependencyHelpers._
 
-  val jacksonVersion = "2.10.2"
-  val junitVersion = "4.13"
+  val jacksonVersion = "2.10.5.1"
```

Comment:
Fossa caught a problem with the old version here: https://github.com/akka/akka-http/runs/4253494683?check_suite_focus=true